.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
```

Comment:
(Maybe as part of the contributing?)

We can maybe add a comment regarding test? I don't think all PR should have test, but core feature, feature that are easy to break, or annoying to manually test would benefit from unit/e2e test